src/org/opendatakit/briefcase/model/FormStatus.java

```diff
@@ -28,6 +28,9 @@
   private String statusString = "";
   private final StringBuilder statusHistory = new StringBuilder();
   private boolean isSuccessful = true;
+  private boolean isPullSelected = false;
```

Comment:
This is a good idea, but this class is used by a myriad of other processes. Could you track the selection of forms in some other way? You could have some new methods and members in `TransferForms`, for example.

Reply:
It's kind of risky changing this form because it belongs to the part of Briefcase that hasn't been refactored yet.